install.sh

```diff
@@ -13,7 +13,11 @@ cp -Rv ./* $DEFAULT_INSTALL_DIR
 GNOME_VERSION=$(gnome-shell --version | awk -F'[ .]' '{print $3}') #get major Gnome version
 if [ $GNOME_VERSION -ge 45 ]; then #apply patch if Gnome 45+
 	echo "You are running GNOME 45 or above. The script will try to patch compatibility for this version. See README.md for details."
-	patch -d $DEFAULT_INSTALL_DIR < patches/gnome45-compatibility.patch && echo "Patch applied!" || echo "Patch failed!" && exit 1
+	patch -d $DEFAULT_INSTALL_DIR < patches/gnome45-compatibility.patch && echo "Gnome 45+ Patch applied!" || { echo "Patch failed!"; exit 1 ; }
```

Comment:
Changing `&&` to `{ ... }` doesn't alter behaviour right?  
If that's the case I do agree that the logic becomes a little bit clearer.

Reply:
Sorry, didn't see the comment. Actually, it does change the way the actions are grouped. With the original implementation, the code was exiting regardless of true/false which meant that it couldn't proceed to the next patch.
- True -> ok message
- False -> failure message
- exit

By grouping using {}, exit only happens with False so:
- True -> ok message -> continue
- False -> failure message -> exit

Reply:
example here, you will see that example #3 doesn't exit as exit it only linked to failure:
![image](https://github.com/user-attachments/assets/d2c8515e-9d7a-4a20-8f54-11d61b5e033f)